<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1301</width>
    <height>923</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>test.jpg</normaloff>test.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="filepath_pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>171</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Directory to Save</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="file_path_lineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>771</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startStim_pushButton">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>260</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>43</red>
          <green>255</green>
          <blue>39</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>43</red>
          <green>255</green>
          <blue>39</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>43</red>
          <green>255</green>
          <blue>39</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>43</red>
          <green>255</green>
          <blue>39</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>43</red>
          <green>255</green>
          <blue>39</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>43</red>
          <green>255</green>
          <blue>39</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>43</red>
          <green>255</green>
          <blue>39</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>43</red>
          <green>255</green>
          <blue>39</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>43</red>
          <green>255</green>
          <blue>39</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(43, 255, 39);</string>
    </property>
    <property name="text">
     <string>Start Experiment</string>
    </property>
   </widget>
   <widget class="GraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>230</y>
      <width>401</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="stim_parameters_groupBox">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>400</y>
      <width>381</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Stim Parameters</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string># Trials done</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select stim</string>
     </property>
    </widget>
    <widget class="QComboBox" name="stim_comboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LeftGrating</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RightGrating</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rivalrous</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="stim_trial_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>311</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>LeftGrating: 0 RightGrating: 0 Rivalrous: 0</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="numframes_spinBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string extracomment="Wait #frames before starting stim"/>
     </property>
     <property name="minimum">
      <number>50</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="singleStep">
      <number>50</number>
     </property>
    </widget>
    <widget class="QLabel" name="wait_frames_label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string extracomment="Wait #frames before starting stim"/>
     </property>
     <property name="text">
      <string>Num Frames</string>
     </property>
    </widget>
    <widget class="QLabel" name="exp_duration_label">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Exp duration: </string>
     </property>
    </widget>
    <widget class="QSlider" name="num_cycles_Slider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="temporal_freq_doubleSpinbox">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <double>2.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="gratings_cycles_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>#Cycles in grating: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Temporal Freq</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="show_stim_checkBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Show Stim</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="camera_parameters_groupBox">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>550</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Right Eye</string>
    </property>
    <widget class="QDoubleSpinBox" name="gain_doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>12.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="gain_slider_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>91</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gain</string>
     </property>
    </widget>
   </widget>
   <widget class="GraphicsView" name="graphicsView1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>401</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="camera1_parameters_groupBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>550</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Left Eye</string>
    </property>
    <widget class="QLabel" name="gain1_slider_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>91</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gain</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="gain1_doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>12.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="exposure_label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>640</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Exposure (us)</string>
    </property>
   </widget>
   <widget class="QLabel" name="framerate_label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>660</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Frame rate =  Hz</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="exposure_spinBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>660</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10000</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1301</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
